README.md

```diff
@@ -41,7 +41,7 @@ Below are the properties allowed by the Changelog Enforcer. These properties are
 
 #### `skipLabels` 
 * Default: `'Skip-Changelog'` 
-* List of labels used to skip enforcing of the changelog during a pull request. Each label name is comma separated and only one label needs to be present for enforcement to be skipped. 
+* List of labels used to skip enforcing of the changelog during a pull request. Each label name is comma separated and only one           label needs to be present for enforcement to be skipped. 
```

Comment:
What's with the whitespace adding on this line? 🤔 Seems like something unintended.